e2e/smartvol_ops_test.go

```diff
@@ -316,6 +317,61 @@ func testSmartVolumeDistributeDisperse(t *testing.T) {
 	checkZeroLvs(r)
 }
 
+func editDevice(t *testing.T) {
+	r := require.New(t)
+	peerList, err := client.Peers()
+	r.Nil(err)
+
+	var deviceList []deviceapi.Info
+	var peerID string
+	for _, peer := range peerList {
+		deviceList, err = client.DeviceList(peer.ID.String())
+		if len(deviceList) > 0 {
+			peerID = peer.ID.String()
+			break
+		}
+	}
+
+	device := deviceList[0]
+	if device.State == "enabled" {
+		err = client.DeviceEdit(peerID, device.Name, "disabled")
+		r.Nil(err)
+	} else if device.State == "disabled" {
+		err = client.DeviceEdit(peerID, device.Name, "enabled")
+		r.Nil(err)
+	}
+	newDeviceList, err := client.DeviceList(peerID)
+	r.Nil(err)
+	for _, newDevice := range newDeviceList {
+		if newDevice.Name == device.Name {
+			r.NotEqual(newDevice.State, device.State)
+		}
+	}
+
+	for _, peer := range peerList {
+		deviceList, err := client.DeviceList(peer.ID.String())
+		r.Nil(err)
+		for _, device := range deviceList {
+			if device.State == "enabled" {
+				err = client.DeviceEdit(peer.ID.String(), device.Name, "disabled")
+				r.Nil(err)
+			}
+		}
+	}
+	smartvolname := formatVolName(t.Name())
+
+	// create Distribute Replicate(2x3) Volume
+	createReq := api.VolCreateReq{
+		Name:               smartvolname,
+		Size:               40 * gutils.MiB,
+		DistributeCount:    2,
+		ReplicaCount:       3,
+		SubvolZonesOverlap: true,
+	}
+	_, err = client.VolumeCreate(createReq)
+	r.NotNil(err)
```

Comment:
enable the devices and create the volume, the volume creation should be successful.